includes/Core.php

```
@@ -271,21 +335,38 @@ private function writeSalts( array $salt_keys, array $new_salts ): bool {
 	private function getConfigFile() {
+		$fs = $this->initFilesystem();
+		if ( ! $fs ) {
+			$this->last_failure_reason     = __( 'Filesystem access is not available.', 'salt-shaker' );
+			$this->last_config_file_path   = '';
+			return false;
+		}
+
 		// Check if the file name is wp-salt.php used in some hosting providers
 		$wp_salts_file   = 'wp-salt';
-		$salts_file_name = ( file_exists( ABSPATH . $wp_salts_file . '.php' ) )
+		$salts_file_name = ( $fs->exists( ABSPATH . $wp_salts_file . '.php' ) )
```

PHPStan (2.2.5)

[error] Constant ABSPATH not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)